recipes-devtools/raspi-gpio/raspi-gpio_git.bb

```diff
@@ -0,0 +1,15 @@
+DESCRIPTION = "Tool to help debug / hack at the BCM283x GPIO"
+HOMEPAGE = "https://github.com/RPi-Distro/raspi-gpio"
+SECTION = "devel/libs"
+LICENSE = "CLOSED"
```

Comment:
The [LICENSE](https://github.com/RPi-Distro/raspi-gpio/blob/master/LICENSE) appears to be [BSD-3-Clause](https://opensource.org/licenses/BSD-3-Clause).

Reply:
@kkoopa ok thanks I'll update PR accordingly

Reply:
Small ping on this @nandra .

Reply:
@agherzan will push shortly